go.py

```diff
@@ -662,7 +651,7 @@ def _removeLinkFromUrls(self, url):
 
     def deleteList(self, LL):
         for link in list(LL.links):
-            L.removeLink(link)
```

Comment:
I've gotta dig deeper about this but I'm curious why this change was made.  Other than this things look fine, I'm currently testing this out with production traffic.

Reply:
`L` is an undefined variable here. `LL` is not. I made that change assuming it was a typo.

Reply:
Ah good to know, I'm amazed it worked all these years, I've got a test server running and am sending production traffic at it and as long as everything looks good with that I'll probably merge this in early next week.